cypress/integration/patch/configure-patch.ts

```diff
@@ -365,8 +365,7 @@ describe("Configure Patch Page", () => {
       );
     });
 
-    // TODO: Reactivate this test when were remove xhr dep on tests
-    xit("Shows error banner if unsuccessful and keeps data", () => {
+    it("Shows error banner if unsuccessful and keeps data", () => {
```

Comment:
This test wont ever work it depends on removing the xhr graphql intercept code from our test suite

Reply:
this test seems to work well in cypress 5. I can remove it if you think it's misleading